creator-node/src/fileManager.js

```diff
@@ -538,6 +556,23 @@ function hasEnoughStorageSpace ({ storagePathSize, storagePathUsed, maxStorageUs
   return (100 * storagePathUsed / storagePathSize) < maxStorageUsedPercent
 }
 
+/**
+ * Remove file from location on disk
+ * @param {String} storagePath path on disk for file
+ * @returns null if successfully removed file or throws error if didn't successfully remove file
+ */
+async function removeFile (storagePath) {
+  try {
+    await fs.unlink(storagePath)
```

Comment:
why fs.unlink instead of fs.rm?

Reply:
they do the same thing under the hood. rm is more heavyweight with more options so decided to keep unlink